src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,290 @@
+authors:
+- bkardell
+reviewers:
+- zcorpan
+- tomhodgins
+- matthewp
+discuss: '1758'
+published: '2019-11-11 00:00:00 +0000'
+last_updated: '2019-11-23 00:00:00 +0000'
+---
+
+## Introduction
+
+En 2005, Ian "Hixie" Hickson a publié [ une analyse des données de balisage ](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) à partir de différents travaux précédents. Une grande partie de ce travail visait à examiner les noms de classes pour voir s’il existait une sémantique informelle commune adoptée par les développeurs et sur laquelle il serait peut-être logique de normaliser. Certaines de ses recherches ont permis d’informer de nouveaux éléments en HTML5.
+
+14 ans plus tard, il est temps de jeter un nouveau coup d’œil. Depuis ce temps, nous avons introduit [ des éléments personnalisés ](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) et l'[ Extensible Web manifesto ](https://extensiblewebmanifesto.org/), ce qui nous a encouragé à trouver des meilleures méthodes pour tracer des chemins en permettant aux développeurs d'explorer l'espace des éléments eux-mêmes et permettent aux organismes de normalisation [ d'agir comme des éditeurs de dictionnaire ](https://bkardell.com/blog/Dropping-The-F-Bomb-On-Standards.html). Par contre les noms de classes CSS, qui peuvent être utilisés pour n'importe quoi, nous pouvons être certains que les auteurs qui ont utilisé un élément {en3} ne sont pas au standard {/en3} ont vraiment voulu qu'il soit un élément.
+
+À partir de juillet 2019, l'archive HTTP a commencé à collecter tous les noms de * éléments * utilisés dans le DOM, environ 4,4 millions de pages d'accueil sur bureau et environ 5,3 millions de pages d'accueil sur mobiles que nous pouvons maintenant commencer à rechercher et à disséquer. * (En savoir plus sur notre [ méthodologie ](./methodology).) *
+
+Cette analyse a rencontré * plus de 5,000 noms d'éléments non standard distincts * dans ces pages. Nous avons donc limité le nombre total d'éléments distincts que nous comptons en 'haut' (expliqué ci-dessous) 5,048.
+
+## Méthodologie
+
+Les noms des éléments sur chaque page ont été collectés à partir du DOM lui même, après l'exécution initiale de JavaScript.
+
+Examiner un nombre de fréquences brut n'est pas particulièrement utile, même pour les éléments standards: environ 25% des éléments rencontrés sont ` div> `. Environ 17% sont ` a> `, environ 11% sont ` span> ` - et ce sont les seuls éléments qui comptent pour plus de 10% des occurrences. Les langues sont [ généralement comme ceci ](https://www.youtube.com/watch?v=fCn8zs912OE); un petit nombre de termes sont étonnamment utilisés par comparaison. En plus, lorsque nous commençons à examiner des éléments non standard pour les adopter, cela peut être très trompeur puisqu'un site pourrait utiliser un certain élément mille fois et lui donner un aspect artificiellement très populaire.
```

Comment:
Appear to have lost some formatting here. Not sure why that is and if you can avoid doing this in future?:

```suggestion
Examiner un nombre de fréquences brut n'est pas particulièrement utile, même pour les éléments standards: environ 25% des éléments rencontrés sont `<div>`. Environ 17% sont `<a>`, environ 11% sont `<span>` -- et ce sont les seuls éléments qui comptent pour plus de 10% des occurrences. Les langues sont [généralement comme ceci](https://www.youtube.com/watch?v=fCn8zs912OE); un petit nombre de termes sont étonnamment utilisés par comparaison. En plus, lorsque nous commençons à examiner des éléments non standard pour les adopter, cela peut être très trompeur puisqu'un site pourrait utiliser un certain élément mille fois et lui donner un aspect artificiellement très populaire.
```

Reply:
> Appear to have lost some formatting here. Not sure why that is and if you can avoid doing this in future?:

because you said do it manually so i switched from Gitlocalize

Reply:
OK but still shouldn't randomly loss `<` symbols and add spacing.

Reply:
> OK but still shouldn't randomly lose `<` symbols and add random spacing? Were these just errors or is it the editor you are using?

just the editor but now all is good